### DISCUSSION TITLE
Ability to log API response body

### DISCUSSION THREAD

**Discussion by kristofre** [Dynatrace/dynatrace-configuration-as-code#194] [Archive] [answered]
We have had some challenges when downloading configuration where it is unable to parse the response body to JSON. It would therefor be great to be able to log the response code and body when supplying the `MONACO_REQUEST_LOG=requests.log` env var.

URL: [url]
METHOD: [method]
CONTENT:
[content]
RESPONSE:
[content]
==========

**Comment by patrickpichler** [comment, marked as answer]
Since we are probably going to run requests in parallel in the near future,
I guess it would make sense to split the response off into its own file.

So you have a `MONACO_RESPONSE_LOG` env var which specifies the
file. Then we only need to generate an UUID so you can match request
and response.

**Reply by patrickpichler** [reply]
I implemented this in #219
